python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -171,10 +171,10 @@ def get_default_workspace(self) -> str:
         )
         return workspaces[0]["workspaceId"]
 
-    def get_source_definition_by_name(self, name: str, workspace_id: str) -> Optional[str]:
+    def get_source_definition_by_name(self, name: str) -> Optional[str]:
         name_lower = name.lower()
         definitions = self.make_request_cached(
-            endpoint="/source_definitions/list_for_workspace", data={"workspaceId": workspace_id}
+            endpoint="/source_definitions/list"
```

Comment:
```suggestion
            endpoint="/source_definitions/list", data={}
```
data param is non-optional at the moment

Reply:
fixed